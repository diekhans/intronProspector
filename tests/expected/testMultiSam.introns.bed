CM000672.2	14299	16501	sj0	2	-	14299	16501	64,128,64
CM000672.2	16544	20785	sj1	2	-	16544	20785	64,128,64
CM000672.2	210048	236837	sj2	4	+	210048	236837	64,128,64
CM000672.2	820845	825064	sj22	1	-	820845	825064	64,128,64
CM000672.2	825097	866693	sj23	1	-	825097	866693	64,128,64
CM000672.2	6226365	6228195	sj3	1	+	6226365	6228195	64,128,64
CM000672.2	6228218	6231262	sj4	4	+	6228218	6231262	64,128,64
CM000672.2	6231314	6232894	sj5	3	+	6231314	6232894	64,128,64
CM000672.2	12167811	12170716	sj6	5	-	12167811	12170716	64,128,64
CM000672.2	12170770	12170899	sj7	5	-	12170770	12170899	64,128,64
CM000672.2	12170908	12172764	sj8	5	-	12170908	12172764	64,128,64
CM000672.2	78037304	78037964	sj24	1	+	78037304	78037964	64,128,64
CM000672.2	78037982	78040203	sj25	1	+	78037982	78040203	64,128,64
CM000672.2	78040225	78040614	sj26	1	+	78040225	78040614	64,128,64
CM000672.2	78040628	78068183	sj27	1	+	78040628	78068183	64,128,64
CM000674.2	56158404	56158683	sj9	1	+	56158404	56158683	64,128,64
CM000674.2	56158711	56158901	sj10	1	+	56158711	56158901	64,128,64
CM000674.2	56158920	56159586	sj11	1	+	56158920	56159586	64,128,64
CM000674.2	56159622	56159974	sj12	1	+	56159622	56159974	64,128,64
CM000674.2	120346390	120357882	sj28	1	-	120346390	120357882	64,128,64
CM000674.2	120357898	120359004	sj29	1	-	120357898	120359004	64,128,64
CM000674.2	120359053	120363042	sj30	1	-	120359053	120363042	64,128,64
CM000675.2	52662702	52663095	sj13	1	+	52662702	52663095	64,128,64
CM000675.2	52663112	52664034	sj14	1	+	52663112	52664034	64,128,64
CM000675.2	75549862	75549975	sj31	1	+	75549862	75549975	64,128,64
CM000675.2	75549987	75554903	sj32	1	+	75549987	75554903	64,128,64
CM000675.2	75554916	75556373	sj33	1	+	75554916	75556373	64,128,64
CM000675.2	75556417	75560752	sj34	1	+	75556417	75560752	64,128,64
CM000676.2	39265421	39276933	sj15	1	+	39265421	39276933	64,128,64
CM000676.2	39276955	39277055	sj16	1	+	39276955	39277055	64,128,64
CM000676.2	39277065	39279336	sj17	1	+	39277065	39279336	64,128,64
CM000676.2	39279358	39279448	sj18	1	+	39279358	39279448	64,128,64
CM000676.2	52658009	52658123	sj35	1	-	52658009	52658123	64,128,64
CM000676.2	52658150	52661292	sj19	4	-	52658150	52661292	64,128,64
CM000676.2	52661304	52663800	sj20	4	-	52661304	52663800	64,128,64
CM000676.2	52663847	52666374	sj21	3	-	52663847	52666374	64,128,64
