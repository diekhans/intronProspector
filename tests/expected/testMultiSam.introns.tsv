chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites	name
CM000672.2	14299	16501	-	0	2	0	12	43	1		sj0
CM000672.2	16544	20785	-	0	2	0	43	47	1		sj1
CM000672.2	210048	236837	+	3	1	0	22	78	2		sj2
CM000672.2	820845	825064	-	0	0	1	46	33	0		sj22
CM000672.2	825097	866693	-	0	0	1	33	14	0		sj23
CM000672.2	6226365	6228195	+	1	0	0	22	23	0		sj3
CM000672.2	6228218	6231262	+	3	0	1	23	52	1.5		sj4
CM000672.2	6231314	6232894	+	2	0	1	52	24	0.918		sj5
CM000672.2	12167811	12170716	-	5	0	0	30	54	1.92		sj6
CM000672.2	12170770	12170899	-	5	0	0	54	9	1.92		sj7
CM000672.2	12170908	12172764	-	5	0	0	9	17	1.92		sj8
CM000672.2	78037304	78037964	+	0	0	1	30	18	0		sj24
CM000672.2	78037982	78040203	+	0	0	1	18	22	0		sj25
CM000672.2	78040225	78040614	+	0	0	1	22	14	0		sj26
CM000672.2	78040628	78068183	+	0	0	1	14	11	0		sj27
CM000674.2	56158404	56158683	+	1	0	0	9	28	0		sj9
CM000674.2	56158711	56158901	+	1	0	0	28	19	0		sj10
CM000674.2	56158920	56159586	+	1	0	0	19	36	0		sj11
CM000674.2	56159622	56159974	+	1	0	0	36	9	0		sj12
CM000674.2	120346390	120357882	-	0	0	1	10	16	0		sj28
CM000674.2	120357898	120359004	-	0	0	1	16	49	0		sj29
CM000674.2	120359053	120363042	-	0	0	1	49	19	0		sj30
CM000675.2	52662702	52663095	+	1	0	0	54	17	0		sj13
CM000675.2	52663112	52664034	+	1	0	0	17	23	0		sj14
CM000675.2	75549862	75549975	+	1	0	0	12	12	0		sj31
CM000675.2	75549987	75554903	+	1	0	0	12	13	0		sj32
CM000675.2	75554916	75556373	+	1	0	0	13	44	0		sj33
CM000675.2	75556417	75560752	+	1	0	0	44	20	0		sj34
CM000676.2	39265421	39276933	+	1	0	0	39	22	0		sj15
CM000676.2	39276955	39277055	+	1	0	0	22	10	0		sj16
CM000676.2	39277065	39279336	+	1	0	0	10	22	0		sj17
CM000676.2	39279358	39279448	+	1	0	0	22	8	0		sj18
CM000676.2	52658009	52658123	-	1	0	0	8	27	0		sj35
CM000676.2	52658150	52661292	-	4	0	0	27	12	0.811		sj19
CM000676.2	52661304	52663800	-	4	0	0	12	47	0.811		sj20
CM000676.2	52663847	52666374	-	3	0	0	47	9	0		sj21
